configs/metrics/otdd.yaml

```diff
@@ -3,14 +3,46 @@ metrics:
   otdd_exact:
     class: otdd
     arguments:
-      max_samples: 1000
-      device: cuda
-      coupling_method: geomloss
+      # general arguments
```

Comment:
could you add top level comments to the args to say they are defaults unless specified and link to the docs?

Reply:
Done!